packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFWebViewFlutterWKWebViewExternalAPI.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <Foundation/Foundation.h>
+#import <WebKit/WebKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+App and package facing native API provided by the `webview_flutter_wkwebview` plugin.
```

Comment:
Nit: Usually our ObjC code uses the:

```
/**
 * Comments here.
 */
```

style for `/* */` comments (i.e., uses the leading `*`).